roles/openshift_logging/tasks/install_logging.yaml

```diff
@@ -4,6 +4,28 @@
     oc_bin: "{{openshift.common.client_binary}}"
     openshift_logging_namespace: "{{openshift_logging_namespace}}"
 
+- conditional_set_fact:
```

Comment:
This section is not very clear what's happening to what.

Reply:
Would adding comments help? The goal here is to reuse variables from a currently deployed configuration as a default rather than a role default when able. So we are pulling out the value from a variable if the variable is defined using the `conditional_set_fact` module from the logging facts. Afterwards we do something similar using the inventory, using `conditional_set_fact` saves us from doing a bunch of `when: <var> is defined` for each possible variable we may do this for